index.js

```diff
@@ -3500,6 +3500,7 @@
 //. [R.equals]:         http://ramdajs.com/docs/#equals
 //. [Ramda]:            http://ramdajs.com/
 //. [RegexFlags]:       https://github.com/sanctuary-js/sanctuary-def#regexflags
+//. [R.map]:            http://ramdajs.com/docs/#map
```

Comment:
Could you insert this line after `[R.equals]` instead? I like to type `vip:sort u` in Vim to sort this "paragraph", so keeping this list sorted will avoid noise in future diffs.